scrapyd_k8s/launcher/k8s.py

```diff
@@ -170,13 +184,12 @@ def _k8s_job_name(self, project, job_id):
 
     def _k8s_kill(self, pod_name, signal):
         # exec needs stream, which modifies client, so use separate instance
-        k8s = kubernetes.client.CoreV1Api()
-        resp = kubernetes.stream.stream(
-            k8s.connect_get_namespaced_pod_exec,
+        kubernetes.stream.stream(
```

Comment:
please keep `resp` here, to remind the reader that it has a result that we may actually want to do something with